Smart.FA.Catalog.IntegrationTests/TrainingContext/User_Tests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Application.UseCases.Queries;
+using AutoFixture;
+using Core.Domain.Enumerations;
+using FluentAssertions;
+using Infrastructure.Services;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using Xunit;
+using Smart.FA.Catalog.IntegrationTests.Base;
+
+namespace Smart.FA.Catalog.IntegrationTests.TrainingContext;
+[Collection("Integration test collection")]
+public class UserTests: IntegrationTestBase
+{
+    private readonly Fixture _fixture = new();
+
+    [Fact]
+    public async Task CanGet()
+    {
+        var logger = Substitute.For<ILogger<GetTrainerFromUserAppQueryHandler>>();
+        await using var context = GivenCatalogContext();
+        var userStrategyResolver = Substitute.For<UserStrategyResolver>("");
+        var handler = Substitute.For<GetTrainerFromUserAppQueryHandler>(logger, context, userStrategyResolver );
+        var request = new GetTrainerFromUserAppRequest {ApplicationType = ApplicationType.Account, UserId = _fixture.Create<string>()};
+
+        var user = await handler.Handle(request, CancellationToken.None);
```

Comment:
Var should be named `trainer` 
```suggestion
        var trainer = await handler.Handle(request, CancellationToken.None);
```

Reply:
Yes good point